### ISSUE TITLE
Tooltips not fully visible in a flex table

### ISSUE THREAD

**Issue by mariodian** [OpenBazaar/OpenBazaar-Client#1361]
Any reason this element is "overflow: hidden"?

```
.flexRow [class |= "flexCol"] {
  box-sizing: border-box;
  position: relative;
  overflow: hidden;
}
```

It breaks tooltips in tables:

![screen shot 2016-04-14 at 7 48 48 pm](https://cloud.githubusercontent.com/assets/3636406/14527089/2d41c17a-027a-11e6-873a-d69033c0ba20.png)

**Comment by Dekker3D** [comment]
It may also be the reason why the "add region" dropdown box gets partially hidden.

**Comment by mariodian** [comment]
Not just "add region" but **any** dropdown box.

![screen shot 2016-04-14 at 8 28 35 pm](https://cloud.githubusercontent.com/assets/3636406/14527977/c37f9fcc-027f-11e6-8551-be1168f1395a.png)

**Comment by jjeffryes** [comment]
Well, that shows me not to skimp on testing.

It was added because the length of the guid sometimes breaks the layout, and as a precaution against other elements doing the same. But clearly not a good idea..

**Comment by mariodian** [comment]
Removing the overflow for the said element and adding these lines

```
    white-space: nowrap;
    overflow: hidden;
    text-overflow: ellipsis;
```

to an element like .fieldItem would truncate long strings as in the screen below.

![screen shot 2016-04-14 at 10 40 33 pm](https://cloud.githubusercontent.com/assets/3636406/14532012/10e79d5c-0292-11e6-8403-fafe913b6e6d.png)

However we should use it with care, because sometimes we may need more lines per a table row. I'd suggest we add a class ".ellipsis" with above properties where we know it may break the layout.

I can create a PR if you're ok with such solution (?).

If an ellipsis is not desirable then we can just use the plain "overflow: hidden". Either way, the overflow shouldn't be set for the parent element (like flexCol-*).

**Comment by jjeffryes** [comment]
This was fixed in #1362. 

There's already a noOverflow class that can be used to selectively add ellipses, it should only be used on divs and spans that contain text.

**Comment by jjeffryes** [comment]
This was fixed in #1362. 

There's already a noOverflow class that can be used to selectively add ellipses, it should only be used on divs and spans that contain text.
